tests/end_to_end/test_netcat/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,61 @@
+- name: Test agent configuration
+  hosts: wazuh-agent
+  tasks:
+
+    - name: Configure agent to get a list of running processes
+      become: true
+      blockinfile:
+        path: /var/ossec/etc/ossec.conf
+        insertbefore: </ossec_config>
+        block: |
+          <localfile>
+          <log_format>full_command</log_format>
```

Comment:
This can be indented as an xml block?

Reply:
no, linter corrected me that everything should be in the same indentation:

- https://github.com/wazuh/wazuh-qa/runs/6919743996?check_suite_focus=true
- https://github.com/wazuh/wazuh-qa/runs/6919979255?check_suite_focus=true